services/cognito/spec/requests/users_spec.rb

```diff
@@ -31,6 +31,38 @@
         expect_json_sizes('data', models_count)
       end
     end
+
+    context 'perform query' do
+      include_context 'authorized user'
+
+      let(:models_count) { rand(1..5) }
+      let!(:models) { create_list :user, models_count }
+
+      context 'based on ID' do
+        before do
+          get url + "?filter[query]=#{models.last.id}", headers: request_headers
```

Comment:
Also, since this will be referenced later on the spec, I'd have a `let(:filtered_model) { models.last }` and then in the url build it using `filtered_model.id`